ERROR_CODES.md

```diff
@@ -178,6 +178,28 @@ public class MyComponent extends ImageView { // Wrong, should extend Parent (or
 show(new MyComponent()); // Wrong, should not be able to show a controller that does not provide a parent as its view
 ```
 
+### 1012: `* '*' in class '*' annotated with '*' cannot be accessed as it is private.`
+
+- Runtime: ✅
+- Annotation Processor: ❌(Caught by the compiler)
```

Comment:
This was only incidentally not removed by the Action script. We should just add a proper diagnostic in the ControllerAnnotationProcessor even if that is somewhat redundant.